code/modules/mob/living/carbon/emote.dm

```diff
@@ -77,10 +95,43 @@
 				'sound/creatures/monkey/monkey_screech_6.ogg',
 				'sound/creatures/monkey/monkey_screech_7.ogg')
 
+/datum/emote/living/carbon/snap
```

Comment:
shouldn't snaps check if you have atleast one usable hand so armless people can't snap?

Reply:
The restraint_check does that I think, but I suppose checking for arm organs could make sense